packages/subgraph/schema.graphql

```diff
@@ -16,6 +16,9 @@ type Global @entity {
   numberOfTrovesClosedByOwner: Int!
   totalNumberOfTroves: Int!
 
+  totalNumberOfLQTYStakes: Int!
+  totalLQTYTokensStaked: BigDecimal!
```

Comment:
We could consider moving `totalLQTYTokensStaked` to `SystemState`, which would allow querying a time series of total stakes. Sure, The Graph lets you go back in time via [time-travel queries](https://thegraph.com/docs/graphql-api#time-travel-queries), but it's pretty awkward to query multiple data points in one query (and makes for a huge query if you want to query for many points).

Reply:
Makes sense, how come we don’t want that for total troves etc?

Reply:
Maybe we do 😉 
We don't have specifications set in stone. We can add stuff as we see fit, or whenever the need arises.